From: David Banks <amoebae@gmail.com>
Date: Sun, 12 May 2013 11:10:44 +0100
Subject: Use the build flags inherited from the environment

The homegrown Makefile overwrites the values of some build flags.  We need the
ability to inject them into the environment from the parent Debian rules file,
so remove this behaviour.

Origin: vendor, Debian
---
 Quake/Makefile |    1 -
 1 file changed, 1 deletion(-)

--- a/Quake/Makefile
+++ b/Quake/Makefile
@@ -52,7 +52,6 @@
 #CPUFLAGS= -mtune=k8
 #CPUFLAGS= -march=atom
 CPUFLAGS=
-LDFLAGS =
 DFLAGS ?=
 CFLAGS ?= -Wall -Wno-trigraphs
 CFLAGS += $(CPUFLAGS)
